FreeBSD/usr/local/share/ceph/example/ceph-build-cluster.sh

```diff
@@ -0,0 +1,654 @@
+#!/bin/sh
```

Comment:
> But you'd still then want to make it `src/script/FreeBSD`, as not to confuse other OSes?

yeah, makes sense to me. or rename this to `src/script/build-cluster.FreeBSD.sh`